Makefile

```diff
@@ -129,7 +139,20 @@ endif
 		$(CR_CONF)=$(RPM_CR_CONF) \
 		conf/clusterrunner-master=/etc/init.d/
 
+.PHONY: docker-rpm
+docker-rpm: TAG=productivity/clusterrunner
```

Comment:
will TAG ever be anything other than productivity/clusterrunner?

Reply:
No, I can declare this in the target, which is probably clearer solution.